students/rmart300/slicing/slicing_methods.py

```diff
@@ -0,0 +1,47 @@
+#slicing methods by Ross Martin for lesson 3 assignment
+
+def exchange_first_last(seq):
+    """exchange first and last elements of a sequence"""
+    return seq[-1:] + seq[1:-1] + seq[:1]
+    
+
+def remove_every_other(seq):
+    """remove every other item from sequence"""
+    return seq[::2]
+
+def remove_first_and_last_four(seq):
+    """remove the first four and the last four of the sequence, return the middle"""
```

Comment:
... and `and then every other item in between`

your assert should be 
`assert remove_first_and_last_four(a_string) == " sas"`